### DISCUSSION TITLE
Odd formatting? Or is it just me?

### DISCUSSION THREAD

**Discussion by ilyvion** [rust-lang/rustfmt#5865] [General]
I just wrote this code, and this is how `rustfmt` decided to format it:
```rust
if piece_place_map
    .get_on_board_team_entities(game_state.current_team_color().unwrap())
    .count()
    == 0
{
    // Victory!
}
```
The thing that throws me off is that `== 0` was put on its own line. Am I the only one who thinks
```rust
if piece_place_map
    .get_on_board_team_entities(game_state.current_team_color().unwrap())
    .count() == 0
{
    // Victory!
}
```
makes more sense?

**Comment by calebcartwright** [comment]
This is correct as per Style Guide Prescription - https://doc.rust-lang.org/nightly/style-guide/expressions.html#binary-operations
